### PR TITLE
Answer to Issue #11 Html to md

### PR DESCRIPTION
**Pull request by ragreenburg**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #(11)

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by ragreenburg** [comment]
Sorry I made a new pull request because I forgot to make a new branch for the first one. So this one has the README, comments and whatnot.

File: HTML_to_MD/html_to_md.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    #converts the html file to a md file
+    md = html2md.convert(html)
+    #Path where your md file is saved
+    md_path = "Path/Where/You/Want/To/Save_MD.file"
+    #Opens the path location and writes from html to md
+    with open(md_path, "w") as mrd:
+        mrd.write(md)
+        mrd.close()
+    #Simply tells you it is done.
+    md_save_str = f"You have successfully written the HTML to a MD file and it is saved at {md_path}"
```
Comment (avinashkranjan) [inline HTML_to_MD/html_to_md.py:21]: Correct Your Syntax..

vmware_9X4xSzdeN7

**Review by avinashkranjan — CHANGES_REQUESTED** [review]
Add Some SS Showing your Script works

**Review by avinashkranjan — APPROVED** [review]

**Comment by avinashkranjan** [comment]
@all-contributors please add @ragreenburg for code and documentation

**Pull request merged**